docs/sources/reference/components/local/local.file_match.md

```diff
@@ -28,6 +28,7 @@ Name            | Type                | Description
 --------------- | ------------------- | ------------------------------------------------------------------------------------------ |---------| --------
 `path_targets`  | `list(map(string))` | Targets to expand; looks for glob patterns on the  `__path__` and `__path_exclude__` keys. |         | yes
 `sync_period`   | `duration`          | How often to sync filesystem and targets.                                                  | `"10s"` | no
+`ignore_older`  | `duration`          | Ignores files which are modified before this duration                                      |         | no
```

Comment:
@wildum If we add that as "0s" wouldn't that be misleading that we ignore files older than current timestamp while the actual implementation considers all the files ?

Reply:
Let's ask the doc expert, @clayton-cornell would you rather:

- have the default value explicitly documented as "0s" with a line under saying that 0 means that it will consider all the files. Although this is more transparent, this might be misleading for users to see the "0s".
- not document the "0s" to implicitly say that this functionality is deactivated by default

or something else?

Reply:
Yup made the change Please take a look